registry/etcdv3/listener_test.go

```diff
@@ -0,0 +1,68 @@
+package etcdv3
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/apache/dubbo-go/common"
```

Comment:
format import

Reply:
Thx, i had reformat import, in latest commit.